test/fixtures/simple-app/common/models/foo-bar-bat-baz-with-def.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (fooBarBatBazWithDef) {
+  fooBarBatBazWithDef.loaded = true;
+};
```

Comment:
As I mentioned above, please rewrite the unit-tests using the newer approach so that application files are created by the test and they are not in fixtures, you can use [this test](https://github.com/shlomiassaf/loopback-boot/blob/d9d0e0297bf8007649778e25729d4b7d8077921c/test/compiler.test.js#L555-L578) as a good example.